services/rollout-service/pkg/service/broadcast.go

```diff
@@ -305,7 +305,7 @@ func (b *Broadcast) Start() ([]*BroadcastEvent, <-chan *BroadcastEvent, unsubscr
 	for key, app := range b.state {
 		result = append(result, app.getEvent(key.Application, key.Environment))
 	}
-	ch := make(chan *BroadcastEvent, 100)
+	ch := make(chan *BroadcastEvent, 100000)
```

Comment:
Couldn't this very high capacity bring memory problems? I'm making the question because there will allways be multiple broadcast channels (not sure how many but at least 3). And this could become a very large load on the rollout service.

I think increasing broadcast channel sizes is a good idea, the number just seems a bit too high. How did you come up with it?

Reply:
There are about 1000 applications and 70 envs, which mean 70000 appEnvs. and at the startup we will have a new message for each of them in the channel. I just assumed the worst case.
With rough estimation, I think it's gonna take 50Mi memory, which I think is ok, but maybe we can decrease it.